server/modules/collectImport/service/collectImport/metaImportJobs/taxonomiesImportJob.js

```diff
@@ -162,8 +161,8 @@ class TaxonomiesImportJob extends Job {
       this.addError({
         [keysError.scientificName]: {
           valid: false,
-          errors: [`duplicate scientific name ${scientificName}; row: ${this.currentRow} duplicate row: ${rowDuplicateScientificName}`]
-        }
+          errors: [{ key: 'duplicateName', params: { scientificName, row: this.currentRow, duplicateRow: rowDuplicateScientificName } }],
```

Comment:
it would be preferable to have duplicateName to keysError constant as the other key errors

Reply:
Yes